test/lib/sstable_utils.hh

```diff
@@ -96,6 +96,8 @@ token_generation_for_shard(unsigned tokens_to_generate, unsigned shard,
 std::vector<std::pair<sstring, dht::token>>
 token_generation_for_current_shard(unsigned tokens_to_generate);
 
+future<compaction_info> compact_sstables(sstables::compaction_descriptor descriptor, column_family& cf, std::function<shared_sstable()> creator, replacer_fn replacer);
```

Comment:
I thought of suggesting defining `replacer = replacer_fn_no_op()` as the default value to save it in most call sites, but you might be against default values.  Up to you...

Reply:
I am not against a default in this case. Suggestion makes sense.